packages/cli/src/lib/snapshot/vaultHandler.ts

```diff
@@ -0,0 +1,176 @@
+import {
+  VaultEntryModel,
+  VaultValueType,
+  VaultVisibilityType,
+} from '@coveord/platform-client';
+import {CliUx} from '@oclif/core';
+import {readJsonSync, rmSync, writeJsonSync} from 'fs-extra';
+import open from 'open';
+import {join} from 'path';
+import {cwd} from 'process';
+import {green, bgWhite} from 'chalk';
+import {
+  InvalidVaultEntryError,
+  InvalidVaultFileError,
+  MissingVaultEntryValueError,
+} from '../errors/vaultErrors';
+import {AuthenticatedClient} from '../platform/authenticatedClient';
+import {Snapshot} from './snapshot';
+import {VaultEntryAttributes} from './snapshotReporter';
+import {Plurable, pluralizeIfNeeded} from '../utils/string';
+import {ProcessAbort} from '../../lib/errors/processError';
+import dedent from 'ts-dedent';
+
+export class VaultHandler {
+  private static readonly defaultEntryValue = '';
+
+  public constructor(private organizationId: string) {}
+
+  public async createEntries(entries: VaultEntryAttributes[]) {
```

Comment:
It's a beefy method that (obviously) execute a lot of stuff... 

Can we try to split it with private functions just for readability sake and see how it looks like ?

pseudo: 
```
public async createEntries(entries: VaultEntryAttributes[]) {
   this.prepareAndOpenFile();
   this.ensureValidityOrAbort();
   this.doCreateVaultEntries();
}